### PR TITLE
Binding Tests

### PR DESCRIPTION
**Pull request by NilKel**
#1152
Binding tests for local coordinate coding.

### CODE REVIEW

**Comment by NilKel** [comment]
@rcurtin review this PR please.

**Comment by manish7294** [comment]
@NilKel Please add the test to CMakeList, otherwise build will not be triggered.

File: src/mlpack/tests/CMakeLists.txt [hunk 1]
```diff
@@ -135,6 +135,7 @@ add_executable(mlpack_test
   main_tests/decision_tree_test.cpp
   main_tests/decision_stump_test.cpp
   main_tests/linear_regression_test.cpp
+  main_tests/local_coordinate_test.cpp
```
Comment (rcurtin) [inline src/mlpack/tests/CMakeLists.txt]: Please note the algorithm is called "local coordinate coding", not "local coordinate".

**Comment by rcurtin** [comment]
I'm going to close this for inactivity.  We can reopen it if desired.

**Pull request closed without merging**